playground/frontend/lib/pages/playground/states/examples_state.dart

```diff
@@ -173,31 +173,27 @@ class ExampleState with ChangeNotifier {
       return;
     }
 
-    List<MapEntry<SDK, ExampleModel>> defaultExamples = [];
-
-    for (var value in SDK.values) {
-      defaultExamples.add(
-        MapEntry(
-          value,
-          await _exampleRepository.getDefaultExample(
-            // First parameter is an empty string, because we don't need path to get the default example.
-            GetExampleRequestWrapper('', value),
-          ),
-        ),
-      );
+    try {
+      await Future.wait(SDK.values.map(_loadDefaultExample));
+    } catch (ex) {
+      if (defaultExamplesMap.isEmpty) {
+        rethrow;
```

Comment:
We used to load the 4 default examples sequentially, and if any of them failed, nothing was loaded. Now go in parallel and allow failures in some of them.